resources/views/meetups/edit.blade.php

```diff
@@ -33,6 +33,7 @@
                         <input type="text" id="language" name="language" value="{{ old('language', $meetup->language) }}">
                         <x-input-error for="language"/>
                     </div>
+                    <img src="{{ asset("storage/" . $meetup->logo) }}" alt="{{ $meetup->title }} logo">
```

Comment:
```suggestion
                    <img src="{{ $meetup->getLogoPath() }}" alt="{{ $meetup->title }} logo">
```
What aboout moving that `asset("storage/" . $meetup->logo)` into model. What do u think?